zenmap/zenmapGUI/ScanInterface.py

```diff
@@ -429,33 +430,46 @@ def start_scan_cb(self, widget=None):
         command = self.command_toolbar.command
         profile = self.toolbar.selected_profile
 
-        log.debug(">>> Start Scan:")
-        log.debug(">>> Target: '%s'" % target)
-        log.debug(">>> Profile: '%s'" % profile)
-        log.debug(">>> Command: '%s'" % command)
+        check = subprocess.Popen(["mktemp"], stdout=subprocess.PIPE).communicate()[0]
```

Comment:
We need to be testing the same mechanism that `zenmapCore.NmapCommand` uses, which is the `tempfile.mktemp` function, not the "mktemp" command (which may not exist on the target system, e.g. Windows).